.github/workflows/build.yml

```
@@ -9,11 +9,39 @@ on:
+      - name: 预先创建release
+        id: create_release
+        uses: ncipollo/release-action@v1
+        with:
+          name: ${{ env.VERSION }}
+          artifacts: ./VERSION
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ env.VERSION }}
+          allowUpdates: true
+          body: ${{ env.VERSION }}
+        env:
+          export PATH: $PATH:/usr/local/go/bin
```

actionlint (1.7.4)

environment variable name "export PATH" is invalid. '&', '=' and spaces should not be contained

(env-var)